### ISSUE TITLE
Remove bundled typescript from packages/server

### ISSUE THREAD

**Issue by sainthkh** [cypress-io/cypress#6051]
### Current behavior:

TypeScript is bundled in `packages/server` because of `dependency-tree`. It takes about 35MB of space. I should be removed to reduce the bundle size.

### Desired behavior:

TypeScript should be removed from the bundle. We need to find ways to replace `dependency-tree`.

### Steps to reproduce: (app code and test code)

`npm ls typescript` in `packages/server`.

### Versions

develop branch.

**Comment by jennifer-shehane** [comment]
@sainthkh is this still relevant?

**Comment by sainthkh** [comment]
Unfortunately, it is.

`@package/server` is still bundling typescript because of `dependency-tree`.

```
dependency-tree@7.0.2
- filing-cabinet@2.3.1
  - typescript@3.0.3
```

**Comment by flotwig** [comment]
@tgriesser closed this in #20905
